tracing-appender/src/lib.rs

```diff
@@ -1,48 +1,127 @@
+//! Writers for logging events and spans
```

Comment:
I would like to wrap some of the long lines in this comment so they don't autowrap in the Github pull review page, but David will disagree with me and I don't really care.

Reply:
I've wrapped the long lines